### DISCUSSION TITLE
tailwind not working with latest vite react and react router installation

### DISCUSSION THREAD

**Discussion by AmitMali** [tailwindlabs/tailwindcss#10008] [Help]
I have done some basic setup to test the react router, installed and configured tailwind as per documentation but it isn't working

i think problem is mainly caused after react router implementation

please check following repository for code

https://github.com/AmitMali/react-router-demo

rechecked all the documentation and implementation

**Comment by wongjn** [comment]
You have some code that uses Tailwind classes but not covered by the `content` file globs. Consider adding `'public/*/*.jsx'` or `'public/**/*.jsx'` to the `content` array in `tailwind.config.js`.

**Comment by dhanushdotcodes** [comment]
bhai did you get the solution
I am facing the same error
